Sources/SwiftKuery/Migration.swift

```diff
@@ -16,51 +16,63 @@
-///
-/// public class MyTable_v0: Table {
-///     let a = Column("a", ...)
-///     let b = Column("b", ...)
-///     let tableName = "MyTable"
-/// }
-///
-/// public class MyTable_v1: Table {
-///     let b = Column("b", ...)
-///     let c = Column("c", ...)
-///     let tableName = "MyTable"
-/// }
-///
-/// And use a typealias to refer to the current version of the table class in the application:
-///
-/// typealias MyTable = MyTable_v0
-/// let t = MyTable()
-/// let q = Select(from t)
-/// ...
-///
-/// The migration code from v0 to v1 should be something like this:
-///
-/// let t0 = MyTable_v0()
-/// let t1 = MyTable_v1()
-/// let migration0 = Migration(from: t0, to: t1, using: connection)
-/// migration0.alterTableAdd(column: t1.c) { result in ... }
-///
-/// And raw alternations, if needed:
-///
-/// let dropColumnQuery = "ALTER TABLE " + t1.tableName + " DROP COLUMN " + t0.a.name
-/// connection.execute(dropColumnQuery) { result in ... }
-///
-
+/**
+ A class to help with migrations between two versions of a table.
+ 
+ ### Usage Example: ###
+ The suggested usage is to keep versions of the table classes somewhere in the application code:
+ ```swift
+ public class MyTable_v0: Table {
+     let a = Column("a", ...)
+     let b = Column("b", ...)
+     let tableName = "MyTable"
+ }
+ public class MyTable_v1: Table {
+     let b = Column("b", ...)
+     let c = Column("c", ...)
+     let tableName = "MyTable"
+ }
+ ```
+ And use a typealias to refer to the current version of the table class in the application:
+ ```swift
+ typealias MyTable = MyTable_v0
+ let t = MyTable()
+ let q = Select(from t)
+ ...
+ ```
+ The migration code from v0 to v1 should be something like this:
+ ```swift
+ let t0 = MyTable_v0()
+ let t1 = MyTable_v1()
+ let migration0 = Migration(from: t0, to: t1, using: connection)
+ migration0.alterTableAdd(column: t1.c) { result in ... }
+ ```
+ And raw alternations, if needed:
+ ```swift
+ let dropColumnQuery = "ALTER TABLE " + t1.tableName + " DROP COLUMN " + t0.a.name
+ connection.execute(dropColumnQuery) { result in ... }
+ ```
+*/
 public class Migration {
     
     private let connection: Connection
     private let table1: Table
     private let table2: Table
     private var renamed = false
     
-    /// Initialize an instance of `Migration`.
-    /// 
+    // MARK: Initializer
+    /**
+     Initialize an instance of `Migration` with the tables you are migrating between
+     and the connection to the database, where the tables are located.
+     ### Usage Example: ###
+     
+     ```swift
+     let connection = getConnection()
+     let oldTable = MyTable_v0()
+     let newTable = MyTable_v1()
+     let migration = Migration(from: oldTable, to: newTable, using: connection)
+     ```
+     */
     /// - Parameter from: The version of the table to migrate from.
     /// - Parameter to: The version of the table to migrate to.
     /// - Parameter using conenction: The connection to the database to use.
```

Comment:
This parameter is missing from the API doc. I think it's because the : is in the wrong place it should be (please fix the spelling of the word connection!
`/// - Parameter connection: The connection to the database to use.`

Reply:
This parameter is still missing from the API doc. It may be that jazzy can't deal with "using connection:", perhaps it'll have to be just "connection:" or "using:"